godotsteam.gdextension

```diff
@@ -9,4 +9,10 @@ windows.release.x86_32 = "bin/libgodotsteam.windows.template_release.x86_32.dll"
 windows.debug.x86_64 = "bin/libgodotsteam.windows.template_debug.x86_64.dll"
 windows.release.x86_64 = "bin/libgodotsteam.windows.template_release.x86_64.dll"
 linux.debug.x86_64 = "bin/libgodotsteam.linux.template_debug.x86_64.so"
-linux.release.x86_64 = "bin/libgodotsteam.linux.template_release.x86_64.so"
\ No newline at end of file
+linux.release.x86_64 = "bin/libgodotsteam.linux.template_release.x86_64.so"
+
+[dependencies]
+macos.debug = { "bin/libsteam_api.dylib": "" }
```

Comment:
I havent looked at GDExtensions myself yet, but the Mac section is for sure wrong, I see it possible to go 2 ways:
Either it'll be `macos.universal` or it'll be `macos.x86_64` and `macos.arm64`